types/ember-cli-mirage/index.d.ts

```diff
@@ -95,7 +95,7 @@ export type Schema = {
 };
 
 export declare class Response {
-    constructor(code: number, headers: Record<string, string>, body: any);
+    constructor(code: number, headers?: Record<string, string>, body?: {});
```

Comment:
Would it be overkill to do `body?: DocWithData | DocWithError` or something like that?

Reply:
The response body could be any object, actually. IMHO, we should leave it at `{}` so we can have mirage handle more than just JSONAPI responses.

Reply:
Sounds good! In that case I think this is good to go 🚀